src/bees.h

```diff
@@ -115,21 +116,25 @@ const int FLAGS_OPEN_FANOTIFY = O_RDWR | O_NOATIME | O_CLOEXEC | O_LARGEFILE;
 
 // macros ----------------------------------------
 
-#define BEESLOG(x)      do { Chatter c(BeesNote::get_name()); c << x; } while (0)
-#define BEESLOGTRACE(x) do { BEESLOG(x); BeesTracer::trace_now(); } while (0)
+#define BEESLOG(l,x)    do { Chatter c(l, BeesNote::get_name()); c << x; } while (0)
+#define BEESLOGTRACE(x) do { BEESLOG(LOG_DEBUG, x); BeesTracer::trace_now(); } while (0)
```

Comment:
This gets called from `trace_now`.  The level should be ERR for consistency (it is only called during exception stack traces).

Reply:
So, I keep this as DEBUG? At least I fixed `l` to not be confused as `1` by renaming it to `lv`.